http/file_server.ts

```diff
@@ -427,11 +427,21 @@ function createBaseHeaders(): Headers {
   });
 }
 
+function html(
+  strings: TemplateStringsArray,
+  ...values: unknown[]
+): string {
+  return strings.reduce(
+    (result, str, i) => result + str + (values[i] ?? ""),
+    "",
+  );
```

Comment:
It looks like MDN recommends implementing this as:
```suggestion
  return String.raw({ raw: strings }, ...values);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw#building_an_identity_tag